documentation/staging/content/faq/external-clients.md

```diff
@@ -141,6 +154,8 @@ Here are the steps:
 
 - __Important__: Ensure that the load balancer configures the HTTP flow to be 'sticky' - for example, a Traefik load balancer has a `sticky sessions` option. This ensures that all of the packets of a tunneling client connection flow to the same pod, otherwise the connection will stall when its packets are load balanced to a different pod.
 
+- __Important__: For EJB and JMS resources that are targeted to a cluster, it is recommended that you configure a default load balancer algorithm that provides server affinity (round-robin-affinity, weight-based-affinity, random-affinity) to enable appropriate routing of requests via replica-aware stubs. For more information see [Load Balancing for EJBs and RMI Objects](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/clust/load_balancing.html#GUID-2470EEE9-F6F9-44EF-BA54-671728E93DE6) in _Load Balancing in a Cluster_.
```

Comment:
"replica-aware stub" is an internal term IMO - so I recommend changing "
to enable appropriate routing of requests via replica-aware stubs" to
something like "to speedup connection creation for EJB and JMS clients."